src/styles.css

```diff
@@ -0,0 +1,165 @@
+
+@import url('https://fonts.googleapis.com/css2?family=Caveat:wght@600&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Poppins&display=swap');
+
+/*////////////////////////
+    Table of Contents
+    1. Global Styles
+    2. Typography
+    3. Layout
+        3.1 Reusables
+        3.2 Other
+    4. Areas
+        4.1 Header
+        4.2 main
+        4.3 country tile
+        etc.
+    5. Media Queries
+////////////////////////*/
+
+/* ------------------------------- 1. Global Styles -------------------------------- */
+
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+body {
+    font-family: 'Roboto', sans-serif;
+    font-weight: 500;
+    font-size: clamp(14px, 1vw, 16px)
```

Comment:
Dit is een schrijfwijze die ik nog nooit gezien heb, ik ga me daar nog eens in verdiepen. Maar de tekst ziet er in ieder geval mooi opgemaakt uit.,

Reply:
> Regel 32: Dit is een schrijfwijze die ik nog nooit gezien heb, ik ga me daar nog eens in verdiepen. Maar de tekst ziet er in ieder geval mooi opgemaakt uit.,

Ja is erg handig voor een responsive webpage. Is volgensmij vrij nieuw en bespaard weer wat media querries.